packages/formation-react/src/components/SortableTable/SortableTable.jsx

```diff
@@ -1,19 +1,64 @@
 import PropTypes from 'prop-types';
-import React from 'react';
+import React, { Component } from 'react';
 import classNames from 'classnames';
 
-class SortableTable extends React.Component {
-  constructor(props) {
-    super(props);
-    this.makeHeader = this.makeHeader.bind(this);
-    this.makeRow = this.makeRow.bind(this);
-  }
+class SortableTable extends Component {
+  static propTypes = {
+    // Used to pass custom classes.
+    className: PropTypes.string,
+
+    /**
+     * Field value to sort by in either ascending or descending order.
+     * The `value` must be one of the values in the `fields` prop.
+     */
+    currentSort: PropTypes.shape({
+      value: PropTypes.string.isRequired,
+      order: PropTypes.oneOf(['ASC', 'DESC']),
+    }).isRequired,
+
+    // Mappings of header labels to properties on the objects in `data`.
+    fields: PropTypes.arrayOf(
+      PropTypes.shape({
+        label: PropTypes.string.isRequired,
+        value: PropTypes.string.isRequired,
+      }),
+    ).isRequired,
+
+    /**
+     * Each object represents data for a row.
+     * An optional class may be provided to style specific rows.
+     */
+    data: PropTypes.arrayOf(
+      PropTypes.shape({
+        id: PropTypes.oneOfType([PropTypes.number, PropTypes.string]).isRequired,
+        values: PropTypes.arrayOf(
+          PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+        ),
+        rowClass: PropTypes.string,
+      }),
+    ).isRequired,
+
+    // A callback for when a header is clicked.
+    onHeaderClick: PropTypes.func,
+
+    // DEPRECATING in favor of onHeaderClick: A callback for when a header is clicked.
+    onSort: PropTypes.func,
+  };
+
+  onHeaderClick = (value, order) => () => {
+    const { onSort } = this.props;
+
+    // This is a legacy prop that is being renamed to `onHeaderClick`.
+    if (onSort) {
+      console.warn('WARNING: The `onSort` prop is being deprecated in favor of the `onHeaderClick` prop for <SortableTable />.');
+      onSort(value, order);
+    }
 
-  handleSort(value, order) {
-    return () => this.props.onSort(value, order);
+    // This replaces `this.props.onSort`.
+    this.props.onHeaderClick(value, order);
```

Comment:
I could be reading it wrong, but do we need an `else` here, so that we don't call both `onSort` and `onHeaderClick`?

Reply:
So I actually thought about returning early on [this line](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/pull/190/files/b674f7b5e5c85017a97495dae15f7d9e43611d38#diff-1ce346f7eeb87e7b181f44b2dabb5566R54) but actually decided against it since there's no errors if they decide to use both in theory! :)